api/tests/unit/infrastructure/open-api-doc/pole-emploi/erreur-doc_test.js

```diff
@@ -0,0 +1,20 @@
+const ErreurDoc = require('../../../../../lib/infrastructure/open-api-doc/pole-emploi/erreur-doc');
+const { expect } = require('../../../../test-helper');
+
+describe('Unit | Infrastructure | Open API Doc | Pole Emploi | Erreur Documentation', () => {
+  it('should validate payload for a campaign participation', () => {
+    // given
+    const payload = {
+      code: 'A1',
+      titre: 'Titre de l\'erreur',
+      statut: '400',
+      detail: 'Description de l\'erreur',
+    };
+
+    // when
+    const result = ErreurDoc.validate(payload);
+
+    // then
+    expect(result.error).to.be.undefined;
```

Comment:
**question** : `to be undefined` ? justement on est pas das le cas d'erreur on donc on veut que ce soit defined ?

Reply:
Non ici on vérifie que la documentation de l'erreur est bonne. 
`result.error` retourne une erreur si la validation n'est pas correcte.

Reply:
ah ok !